tower/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 # Unreleased
 
 - **util**: Add `CloneBoxService` which is a `Clone + Send` boxed `Service`.
+- **util:** Add `ServiceExt::boxed` for applying a `BoxService` middleware.
+- **util:** Add `ServiceExt::clone_boxed` for applying a `CloneBoxService` middleware.
+- **builder:** Add `ServiceExtBuilder::boxed` for applying a `BoxService` layer.
+- **builder:** Add `ServiceExtBuilder::clone_boxed` for applying a `CloneBoxService` layer.
```

Comment:
take it or leave it: i think it would probably be fine to combine some of these changelog entries, like
```suggestion
- **util:** Add `ServiceExt::boxed` and `ServiceExt::clone_boxed` for applying the
   `BoxService` and `CloneBoxService` middleware.
- **builder:** Add `ServiceBuilder::boxed` and `ServiceBuilder::clone_boxed` for applying 
   `BoxService` and `CloneBoxService` layers.
```

Reply:
also, i noticed these changelog entries refer to `ServiceExtBuilder`, which we should fix regardless of whether we apply my suggestion to combine the entries:
```suggestion
- **util:** Add `ServiceExt::boxed` for applying a `BoxService` middleware.
- **util:** Add `ServiceExt::clone_boxed` for applying a `CloneBoxService` middleware.
- **builder:** Add `ServiceBuilder::boxed` for applying a `BoxService` layer.
- **builder:** Add `ServiceBuilder::clone_boxed` for applying a `CloneBoxService` layer.
```